CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ UNRELEASED
 * [ [#522](https://github.com/digitalfabrik/integreat-cms/issues/522) ] Add region setting to activate SEO section, add SEO section to POI form
 * [ [#1858](https://github.com/digitalfabrik/integreat-cms/issues/1858) ] Enable submitting feedback about fallback translations of recurring events
 * [ [#1865](https://github.com/digitalfabrik/integreat-cms/issues/1865) ] Fix TextLab HIX widget for non-staff users
+* [ [#1788](https://github.com/digitalfabrik/integreat-cms/issues/1788) ] Fix broken translation status of events & locations if only minor public versions exists
```

Comment:
Could you put this entry into the "unreleased" section?

Reply:
Oh shit, yes of course